ci/fireci/fireci/dir_utils.py

```diff
@@ -30,3 +33,28 @@ def chdir(directory):
   finally:
     _logger.debug(f'Restoring directory to: {original_dir} ...')
     os.chdir(original_dir)
+
+def rmdir(path: str) -> bool:
+  """Recursively deletes a directory, and returns a boolean indicating if the dir was deleted."""
+  dir = pathlib.Path(path)
+  if not dir.exists():
+    _logger.debug(f"Directory already deleted: {dir}")
+    return False
+
+  _logger.debug(f"Deleting directory: {dir}")
+  shutil.rmtree(dir)
+  return True
+
+def rmglob(pattern: str) -> int:
+  """Deletes all files that match a given pattern, and returns the amount of (root) files deleted"""
+  files = 0
+  for file in glob.glob(os.path.expanduser(pattern)):
+    path = pathlib.Path(file)
+    if path.is_dir():
+      rmdir(file)
+    else:
+      _logger.debug(f"Deleting file: {path}")
+      os.remove(path)
+    files += 1
+
+  return files
```

Comment:
Out of curiosity, why you need the count?

Reply:
To display at the end of the `clean` run. It makes it easier to see the result if you have `debug` logging disabled.

![example](https://i.imgur.com/lIbLY3U.png)